Catappult/appcoins-wallet-android
# [APPC-4170] Hotfix - Remove ask notification permission on onboarding

**What does this PR do?**

  Remove ask notification permission from onboarding payment and unused code of referral feature

**Database changed?**

  No


**How should this be manually tested?**

  verify if ask notification permission is showed when enter normally on app and not showed on onboarding flow

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4170](https://aptoide.atlassian.net/browse/APPC-4170)

**Code Review Checklist**



[APPC-4170]: https://aptoide.atlassian.net/browse/APPC-4170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
